### PR TITLE
Feature/mobile header optimization

### PR DESCRIPTION
**Pull request by priyankatechneai**
Header optimization

### CODE REVIEW

File: src/components/Common/Header.js
Comment (AmitSolnke) [inline src/components/Common/Header.js:1]: Tablet view: It is not responsive yet.

**Review by AmitSolnke — CHANGES_REQUESTED** [review]
1.remove rebundant code .(remove code which are not used).
2.Apis are not been checked for error.If api crashes our client side system will also get crashed.Instead show user friendly error to user.
3.const [notify, setNotify] = useState(null); Its good practice to specify date iniatially to avoid unwanted error while rendering.
4.Remove console log (Strict warning)

File: src/components/Common/Header.js [hunk 1]
```diff
@@ -44,7 +37,7 @@ export default function Header() {
   const [showApprovedOnly, setShowApprovedOnly] = useState(false);
 
   const loadNotifcation = () => {
-    getNotification().then(res => {
+    getNotification().then((res) => {
```
Comment (AmitSolnke) [inline src/components/Common/Header.js:40]: 1. There is no async await.
2.No optional chaining.
3.

File: src/components/Common/Header.js [hunk 2]
```diff
@@ -44,7 +37,7 @@ export default function Header() {
...
       if (res.status === 200) {
```
Comment (AmitSolnke) [inline src/components/Common/Header.js:41]: 1.Not checked for res?.data?.status === 1

File: src/components/Common/Header.js [hunk 3]
```diff
@@ -53,10 +46,14 @@ export default function Header() {
             var length = res.data.data.result.length;
             var height = 0;
             setNotifications(res.data.data.result);
-            // setApprovedNotifications(res.data.data.for_me);
-            setApprovedNotifications(res?.data?.data?.result?.filter(d => d?.status == 1));
 
-            setAllRequest(res?.data?.data?.result?.filter(d => d?.status != 1));
+            setApprovedNotifications(
```
Comment (AmitSolnke) [inline src/components/Common/Header.js:50]: 1.Rather implementing filter implicitly in setstate , Have a habbit of storing operation in variable and than pass it to setstate.

File: src/components/Common/Header.js
Comment (AmitSolnke) [reply src/components/Common/Header.js:1]: 1.  handleCloseApproveRequestModal & handleCloseHistoryModal are not used in code anywhere,
2.Inconsistent condition , at some places res.status === 200 is used and at some places res.data.status === 1 is used.
3.In Dropdown.menu key is not provided in li tag.

File: src/components/Common/Header.js (hunk 3 above)
Comment (priyankatechneai) [reply src/components/Common/Header.js:50]: resolved

File: src/components/Common/Header.js [hunk 4]
```diff
@@ -53,10 +46,14 @@ export default function Header() {
             var length = res.data.data.result.length;
```
[the remark this answers is not in the document]
Comment (priyankatechneai) [reply src/components/Common/Header.js:46]: resolved

File: src/components/Common/Header.js (hunk 2 above)
Comment (priyankatechneai) [reply src/components/Common/Header.js:41]: resolved

File: src/components/Common/Header.js
Comment (priyankatechneai) [reply src/components/Common/Header.js:1]: resolved

File: src/components/Common/Header.js (hunk 1 above)
Comment (priyankatechneai) [reply src/components/Common/Header.js:36]: resolved

File: src/screens/TicketManagement/MyTicketComponent.jsx [hunk 5]
```diff
@@ -274,12 +277,39 @@ export default function MyTicketComponent() {
...
               </li>
 
-              {data.created_by != localStorage.getItem("id") &&
+              {/* {data.created_by != localStorage.getItem("id") &&
                 data.basket_configured === 0 &&
                 localStorage.getItem("account_for") === "SELF" &&
                 data.status_name != "Solved" &&
                 data.passed_status !== "REJECT" &&
-                data.passed_status !== "UNPASS" && (
+                data.passed_status !== "UNPASS" && ( */}
```
Comment (somyaranjan-techneai) [inline src/screens/TicketManagement/MyTicketComponent.jsx:285]: if it's not req in the feature remove commented code,
 if req add comment in the code

File: src/screens/TicketManagement/MyTicketComponent.jsx [hunk 6]
```diff
@@ -307,7 +335,9 @@ export default function MyTicketComponent() {
                       <i className="icofont-tasks"></i> Task
                     </Link>
```
Comment (somyaranjan-techneai) [inline src/screens/TicketManagement/MyTicketComponent.jsx:336]: make self-closing tags

File: src/screens/TicketManagement/MyTicketComponent.jsx [hunk 7]
```diff
@@ -318,6 +348,18 @@ export default function MyTicketComponent() {
                   <i className="icofont-history"></i> History
```
Comment (somyaranjan-techneai) [inline src/screens/TicketManagement/MyTicketComponent.jsx:348]: make self-closing tags

**Pull request closed without merging**